slicer-plugin/NvidiaAIAA/SegmentEditorNvidiaAIAALib/SegmentEditorEffect.py

```diff
@@ -512,30 +531,43 @@ def onClickDeepgrow(self, current_point):
         if in_file is None and session_id is None:
             return
 
+        # use model name to determine
+        deepgrow_3d = True if '3d' in model.lower() else False
```

Comment:
Have updated the AIAA to support additional info which will be passed in model details. Lets not assume from model name if it's 3D.. rather from the model details (attribute "deepgrow": "3D")

{
name: "deepgrow_3d",
labels: [ ],
description: "3D DeepGrow model based on Unet",
version: "3",
type: "deepgrow",
deepgrow: "3D"
}

Reply:
fixed